src/jdk.compiler/share/classes/com/sun/tools/javac/util/Log.java

```diff
@@ -312,11 +312,6 @@ protected Log(Context context, PrintWriter out, PrintWriter err) {
 
     /**
      * Construct a log with given I/O redirections.
-     * @deprecated
-     * This constructor is provided to support the supported but now-deprecated javadoc entry point
-     *      com.sun.tools.javadoc.Main.execute(String programName,
-     *          PrintWriter errWriter, PrintWriter warnWriter, PrintWriter noticeWriter,
-     *          String defaultDocletClassName, String... args)
```

Comment:
It is generally bad form to have `@Deprecated` without `@deprecated`. In this case, separate ongoing work to cleanup `Messager` will eventually mean we can remove the method altogether.

If you really want to remove the reference to the old code, the `@deprecated` comment should be updated to "Still in use by javadoc's Mesager class." or some such.